pfcwd/main.py

```diff
@@ -272,15 +276,23 @@ def start_cmd(self, action, restoration_time, ports, detection_time):
         for port in ports:
             if port == "all":
                 for p in all_ports:
+                    pfc_status = self.get_port_pfc_status(p)
+                    if pfc_status == None:
```

Comment:
status can be None or 'disable'. I suggest this function will return true/false and will return true only if status == 'enabled'